lms/djangoapps/ora_staff_grader/serializers.py

```diff
@@ -150,8 +152,13 @@ class GradeDataSerializer(serializers.Serializer):
 
 class SubmissionDetailResponseSerializer(serializers.Serializer):
     """ Serializer for the response from the submission """
-    gradeData = GradeDataSerializer(source='assessment')
-    response = ResponseSerializer(source='submission')
-    #  to be completed in AU-387
-    #  gradeStatus = GradeStatusField()
-    #  lockStatus = LockStatusField()
+    gradeData = GradeDataSerializer(source='submission_and_assessment_info.assessment')
+    response = ResponseSerializer(source='submission_and_assessment_info.submission')
+    gradeStatus = serializers.SerializerMethodField()
```

Comment:
This would be a hair simpler as `source=grade_and_submission_info.assessment` but SerializerMethodField doesn't support `source`
 
 https://github.com/encode/django-rest-framework/blob/060a3b632f6f6ff2f84235d1be5da55020c40ff3/rest_framework/fields.py#L1871